editor/src/clj/editor/tile_source.clj

```diff
@@ -629,16 +633,14 @@
   (output tile-source-attributes g/Any :cached produce-tile-source-attributes)
   (output tile->collision-group-node g/Any :cached produce-tile->collision-group-node)
 
-  (output texture-set-data-generator g/Any (g/fnk [image-resource tile-source-attributes animation-data collision-groups convex-hulls tile-count :as args]
+  (output texture-set-data-generator g/Any (g/fnk [_node-id image-resource tile-source-attributes animation-data collision-groups convex-hulls tile-count :as args]
                                              (or (when-let [errors (not-empty (mapcat #(check-anim-error tile-count %) animation-data))]
                                                    (g/error-aggregate errors))
                                                  (let [animation-ddfs (mapv :ddf-message animation-data)]
                                                    {:f generate-texture-set-data
                                                     :args (-> args
                                                               (dissoc :animation-data)
-                                                              (assoc
-                                                                :animation-ddfs animation-ddfs
-                                                                :image-resource image-resource))}))))
+                                                              (assoc :animation-ddfs animation-ddfs))}))))
```

Comment:
We're already getting the `:image-resource` as part of `args`, so no need to do `assoc` it here.